src/Util/util.cpp

```diff
@@ -505,11 +509,15 @@ string getThreadName() {
     string ret;
     ret.resize(32);
     auto tid = pthread_self();
+#ifndef pthread_getname_np
```

Comment:
Any reference for this macro `pthread_getnanme_np`?

Reply:
Thanks for the reply, it was a mistake.